README.md

```diff
@@ -132,7 +132,18 @@ We'll be updating this once every month or so as we complete new topics in our o
 * [Dagger 2: A New Type of Dependency Injection](https://www.youtube.com/watch?v=oK_XtfXPkqw)
 * [Retrofit 2](https://www.youtube.com/watch?v=KIAoQbAu3eA)
 * [Android Tech Talk: Annotation Processing Boilerplate Destruction](http://www.youtube.com/watch?v=dOcs-NKK-RA)
+
+#### Making Your App Work Offline
 * [Realm - A New and Easy to use Object Framework and Database](https://www.youtube.com/watch?v=QT7XD1hifkU)
+* [Rule Your Realm](https://www.youtube.com/watch?v=ZTzg6lOgYiI)
+* [Android Architecture for the Subway](https://academy.realm.io/posts/360-andev-2017-ben-oberkfell-android-architecture-offline-first/)
+* [Don’t Fear SQL: Store and Handle Data with SQLBright and SQLDelight](https://academy.realm.io/posts/360-andev-2017-leandro-favarin-sqlbrite-sqdelight/)
+
+#### Android App Architecture
+* [Architecture Components - Introduction](https://www.youtube.com/watch?v=FrteWKKVyzI&list=PLWz5rJ2EKKc-odHd6XEaf7ykfsosYyCKp&index=4) 
```

Comment:
For this and other links, could link to just the v= part:
https://www.youtube.com/watch?v=FrteWKKVyzI

But nbd